bentoml/bentos.py

```diff
@@ -85,56 +85,51 @@
 from fs.mirror import mirror
 from simple_di import Provide, inject
 
-from ._internal.bento import Bento
+from ._internal.bento import Bento, BentoStore
 from ._internal.configuration.containers import BentoMLContainer
 from ._internal.types import Tag
 from ._internal.utils import generate_new_version_id
 
 if t.TYPE_CHECKING:
     from ._internal.models.store import ModelStore
     from ._internal.service import Service
-    from ._internal.store import Store
 
 from ._internal.service import load
 
 
 @inject
 def list(
     tag: t.Optional[t.Union[Tag, str]] = None,
-    _bento_store: "Store" = Provide[BentoMLContainer.bento_store],
+    _bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
 ) -> t.List[Tag]:
     return _bento_store.list(tag)
 
 
 @inject
 def get(
     tag: t.Union[Tag, str],
-    _bento_store: "Store" = Provide[BentoMLContainer.bento_store],
+    _bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
 ) -> Bento:
     bento_fs = _bento_store.get(tag)
-    return Bento.import_from_fs(bento_fs)
+    return Bento.from_fs(bento_fs)
 
 
 def delete(
     tag: t.Union[Tag, str],
-    _bento_store: "Store" = Provide[BentoMLContainer.bento_store],
+    _bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
 ):
     _bento_store.delete(tag)
 
 
 def import_bento(path: str) -> Bento:
-    return Bento.import_from_fs(fs.open_fs(path))
+    return Bento.from_fs(fs.open_fs(path))
```

Comment:
probably for future PR, here we may need to choose the fs type based on the path or extenstion